data/Boardmember.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "name": "Fidal Mathew",
+    "image": "https://avatars.githubusercontent.com/u/84982038?v=4",
+    "role": "Open Source Lead",
+    "description": "Web Developer | Learning :)",
+    "github": "https://github.com/FidalMathew",
+    "linkedIn": "https://www.linkedin.com/in/fidalmathew/",
+    "color": "#4b916c"
+  },
+  {
+    "name": "Nimish Kashyap",
```

Comment:
He is not the blockchain lead please chain the data, or add other people whom you know